docs/api-docs/users/overview.mdx

```diff
@@ -905,6 +906,45 @@ The following mutation adds a user to a store:
   </Tab>
 </Tabs>
 
+Example using the user's email to add the user to the store:
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+
+    ```graphql filename="Example mutation: Add user to store" showLineNumbers copy
+    mutation {
+      user {
+        addUserToStore(
+          input: {
+            user: {
+              email: "bc/account/user/{user_email}"
+            },
+            storeId:"bc/account/store/{store_id}"
+          }
+        ) {
+          storeId
+        }
+      }
+    }
+    ```
+
+  </Tab>
+  <Tab>
+
+    ```json filename="Example mutation: Add user to store" showLineNumbers copy
+    {
+      "data": {
+        "user": {
+          "addUserToStore": {
+            "storeId": "bc/account/store/{store_hash}"
```

Comment:
are we really doing store hash here for `storeId` and `store_id` in the input?  ew.

Reply:
Yes, we are indeed. It's not great